documentation/JFrog-CLI.md

```diff
@@ -329,13 +329,19 @@ The **config add** and **config edit** commands are used to add and edit JFrog P
 | --user                 | \[Optional\]<br><br>JFrog Platform username.                                                                                                                                                                                                                                                                                                                                                   |
 | --xray-url             | \[Optional\] Xray URL.                                                                                                                                                                                                                                                                                                                                                                         |
-| --overwrite            | \[Available for _config add_ only\]<br><br>\[Default: false\]<br><br>Overwrites the instance configuration if an instance with the same ID already exists.                                                                                                                                                                                                                                     |
+| --overwrite            | \[Available for `config add` only\]<br><br>\[Default: false\]<br><br>Overwrites the instance configuration if an instance with the same ID already exists.                                                                                                                                                                                                                                     |
 | Command arguments      |                                                                                                                                                                                                                                                                                                                                                                                                |
 | server ID              | A unique ID for the server configuration.                                                                                                                                                                                                                                                                                                                                                      |
 
+### Configuring using Web Login
+
+The `login` command is used to add a JFrog Platform server configuration, by logging in to the platform via your web browser.
+This command is solely interactive, meaning it does not receive any options and cannot be used in a CI server.
```

Comment:
1. 
I suggest rephrasing the above section as follows:

### Web Login to the JFrog Platform
You can use the `jf login` command to authenticate with the JFrog Platform through the web browser. This command is solely interactive, meaning it does not receive any options and cannot be used in a CI server.

2.
Let's also put this section before the **Adding and Editing Configured Servers**, as the first authentication option.

3.
Let's also put add two screenshots from the login page.